public/components/security/roles-mapping/helpers/rule-editor.helper.test.ts

```diff
@@ -201,4 +201,159 @@ describe('Rule Editor Helper', () => {
       });
     });
   });
-});
+
+  describe('Test role-mapping behavior', () => {
+    describe('Given a non null rules array and non null internal users array ', () => {
```

Comment:
Here the title says that rules and internal users are null, but they are not. They are correct? Or maybe you can copy this test and create a new one with null values and cover this case 😄 

Great job!